test/hotspot/jtreg/runtime/CompressedOops/CompressedClassPointersEncodingScheme.java

```diff
@@ -101,25 +101,27 @@ public static void main(String[] args) throws Exception {
         // add more...
 
         // Compact Object Header Mode:
-        // On all platforms we expect the VM to chose the smallest possible shift value needed to cover
+        // On aarch64 and x64 we expect the VM to chose the smallest possible shift value needed to cover
         // the encoding range. We expect the encoding Base to start at the class space start - but to enforce that,
         // we choose a high address.
-        long forceAddress = 32 * G;
+        if (Platform.isAArch64() || Platform.isX64()) {
```

Comment:
@rkennke please also enable riscv for this test `CompressedClassPointersEncodingScheme.java`, it passed in my environment. Thanks!

Reply:
Done.